docs/system/system-metrics.md

```diff
@@ -1466,6 +1467,25 @@ See also `MemAvailable` in [/proc/meminfo](https://man7.org/linux/man-pages/man5
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `system.linux.memory.slab`
+
+<!-- semconv metric.system.linux.memory.slab(metric_table) -->
```

Comment:
as your metric has attributes they should be noted here i.e. add after metric table
```
<!-- semconv metric.system.linux.memory.slab(full) -->
```

Reply:
I guess this can be resolved now.